homeassistant/components/notify/__init__.py

```diff
@@ -185,6 +198,14 @@ async def async_register_services(self) -> None:
                     self._async_notify_message_service,
                     schema=NOTIFY_SERVICE_SCHEMA,
                 )
+                # Register the service description
+                service_desc = {
+                    CONF_DESCRIPTION: f"Send a notification to {target_name}",
+                    CONF_FIELDS: self.services_dict.get(SERVICE_NOTIFY, {}).get(
+                        CONF_FIELDS, {}
+                    ),
```

Comment:
We know the entry exists.

```suggestion
                    CONF_FIELDS: self.services_dict[SERVICE_NOTIFY][CONF_FIELDS],
```